ModularTegustation/tegu_items/lc13/refinery/crates.dm

```diff
@@ -0,0 +1,70 @@
+/obj/structure/lootcrate
+	name = "Crate"
+	desc = "A crate recieved from a company"
+	icon = 'ModularTegustation/Teguicons/refiner.dmi'
+	icon_state = "crate_lcb"
+	anchored = FALSE
+	density = TRUE
+	resistance_flags = INDESTRUCTIBLE
+	var/lootlist =	list(
+		/obj/item/toy/plush/beeplushie, //higher values are more common
+		/obj/item/toy/plush/blank,
+		/obj/item/toy/plush/yisang,
+		/obj/item/toy/plush/faust,
+		/obj/item/toy/plush/don,
+		/obj/item/toy/plush/ryoshu,
+		/obj/item/toy/plush/meursault,
+		/obj/item/toy/plush/honglu,
+		/obj/item/toy/plush/heathcliff,
+		/obj/item/toy/plush/ishmael,
+		/obj/item/toy/plush/rodion,
+		/obj/item/toy/plush/sinclair,
+		/obj/item/toy/plush/outis,
+		/obj/item/toy/plush/gregor,
+		/obj/item/toy/plush/dante,
+		/obj/item/toy/plush/yuri)
+
+/obj/structure/lootcrate/attackby(obj/item/I, mob/living/user, params)
+	..()
+	if(I.tool_behaviour == TOOL_CROWBAR)
+		var/loot = pick(lootlist)
+		to_chat(user, "<span class='notice'>You open the crate!</span>")
+		new loot(get_turf(src))
+		qdel(src)
+
+/obj/structure/lootcrate/lingus
+	name = "Limbus Company Crate"
+	desc = "A crate recieved from limbus company. Open with a Crowbar."
+	icon_state = "crate_lcb"
+	lootlist =	list(
+		/obj/item/ego_weapon/mini/hayong,
+		/obj/item/ego_weapon/shield/walpurgisnacht,
+		/obj/item/ego_weapon/lance/suenoimpossible,
+		/obj/item/ego_weapon/shield/sangria,
+		/obj/item/ego_weapon/mini/soleil,
+		/obj/item/ego_weapon/taixuhuanjing,
+		/obj/item/ego_weapon/revenge,
+		/obj/item/ego_weapon/shield/hearse,
+		/obj/item/ego_weapon/mini/hearse,
+		/obj/item/ego_weapon/raskolot,
+		/obj/item/ego_weapon/vogel,
+		/obj/item/ego_weapon/nobody,
+		/obj/item/ego_weapon/ungezifer,
+		/obj/item/clothing/suit/armor/ego_gear/limbus/limbus_coat,
+		/obj/item/clothing/suit/armor/ego_gear/limbus/limbus_coat_short,
+		/obj/item/clothing/under/limbus/shirt,
+		/obj/item/clothing/accessory/limbusvest,
+		/obj/item/clothing/under/limbus/prison,
+		/obj/item/clothing/neck/limbus_tie,)
+
+/obj/structure/lootcrate/kcorp
```

Comment:
```suggestion
/obj/structure/lootcrate/k_corp
```
Not a strict requirement, but it's prefered to avoid merging words in path names.